.github/workflows/generate-changelog.yaml

```diff
@@ -0,0 +1,127 @@
+name: Generate changelog
+# Reusable workflow to generate changelog after release
```

Comment:
I think the name should be kept short and descriptive because it can appear in the GitHub UI and sadly a proper description field is not supported. So I prefer having a comment for the description